### PR TITLE
Dev memberproject

### PR DESCRIPTION
**Pull request by tuanhai2712**

### CODE REVIEW

File: app/Http/Controllers/MemberProjectController.php [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    {
+        $listProjectAssigned = Member::find($id)->project;
+        //return view keep for next test app
+    }
+
+    public function addParticipants($idMember, $idProject)
+    {
+        $findMember = Member::find($idMember);
+        $findProject = Project::find($idProject);
+        if ($findProject->member()->contains($findMember)) {
```
Comment (TaHieu-scuti) [inline app/Http/Controllers/MemberProjectController.php]: I dont see any thing in this "IF". Pls remove it if you dont need them.

File: app/Http/Controllers/MemberProjectController.php [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+            return 'Participant already exist';
+        }
+        $addParticipants = $findProject->member()->attach($findMember);
+            return 'Add participants success'
```
Comment (TaHieu-scuti) [inline app/Http/Controllers/MemberProjectController.php]: indentation wrong.! Pls fix this.

File: app/Member.php [hunk 3]
```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Member extends Model
+{
```
Comment (ngon12) [inline app/Member.php]:
```
protected $fillable = [
    'member_name', 'member_info',
    'member_phone', 'member_dateofbirth',
    'member_avatar', 'member_position', 'member_gender',
];
```

Please format like that. Thanks

File: app/MemberProject.php [hunk 4]
```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class MemberProject extends Model
+{
+    //
```
Comment (ngon12) [inline app/MemberProject.php]: Remove, please! Thanks

File: app/Project.php [hunk 5]
```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Project extends Model
+{
+    protected $fillable = ['project_name', 'project_info', 'project_deadline', 'project_type', 'project_status'];
```
Comment (ngon12) [inline app/Project.php]: Please format it like that: 

```
protected $fillable = [
    'project_name', 'project_info', 
    'project_deadline', 'project_type', 
    'project_status'
];
```

File: routes/web.php [hunk 6]
```diff
@@ -13,4 +13,4 @@
 
 Route::get('/', function () {
     return view('welcome');
-});
+});
```
Comment (ngon12) [inline routes/web.php]: Please add more line to this. Thanks

**Pull request merged**